AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -0,0 +1,93 @@
+// Created by Lunar on 16/02/2022.
+//
+
+import CoreLocation
+import Foundation
+import SwiftUI
+import AirCastingStyling
+
+struct SearchMapView: View {
+    @StateObject var viewModel: SearchMapViewModel
+    @Environment(\.presentationMode) var presentationMode
+    
+    init(locationName: String, locationAddress: CLLocationCoordinate2D, parametr: String) {
```

Comment:
typo in `parameter`. Also the name `parameter` is not saying anything, please rename